libcaf_test/caf/test/test.test.cpp

```diff
@@ -40,6 +41,51 @@ TEST("tests can contain different types of checks") {
   info("this test had {} checks", rep.test_stats().total());
 }
 
+#ifdef CAF_ENABLE_EXCEPTIONS
+
+TEST("tests fail when requirement errors occur") {
+  auto& rep = caf::test::reporter::instance();
```

Comment:
We are using `caf::test::requirement_failed` quite a lot, adding a `using` here allows us to reduce the verbosity below.

```suggestion
TEST("tests fail when requirement errors occur") {
  using caf::test::requirement_failed;
  auto& rep = caf::test::reporter::instance();
```